duradmin/src/main/webapp/jquery/dc/widget/ui.snapshot.js

```diff
@@ -112,8 +112,25 @@ $.widget("ui.snapshot",
                                                     textStatus, 
                                                     errorThrown ) {
                                         dc.done();
-                                        dc.displayErrorDialog(jqXHR, 
-                                                              SnapshotErrorMessage.UNAVAILABLE, 
+
+                                        var message = SnapshotErrorMessage.UNAVAILABLE;
+
+                                        var responseText = jqXHR.responseText;
+
+                                        if(jqXHR.status == 409){
+                                        	message = responseText;
+										}
+
+                                        if(responseText.startsWith("{")){
+                                        	var json = JSON.parse(responseText);
+                                        	if(json['exception.message']){
+                                                message = json['exception.message'];
+											}
+										}
+
+
+                                        dc.displayErrorDialog(jqXHR,
+                                                              message,
```

Comment:
The formatting of the code added in this file looks a bit... odd (e.g. lots of spaces between lines, brackets don't appear to be lined up, odd wrapping)